zerver/two_factor_migrations/0001_initial.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+from django.conf import settings
+import django.core.validators
```

Comment:
I don't understand why this is necessary -- is it a bug in the `two_factor` app that we're working around?  It seems to me that `django_two_factor` should be creating the migrations that it needs automatically... I'd rather we investigate and figure out why that isn't happening so that we can use migrations from the `two_factor` app (since I worry that this setup will be hard to clean up later when `two_factor` does get fixed).